source/D2Common/src/DataTbls/HoradricCube.cpp

```diff
@@ -300,6 +300,7 @@ BOOL __fastcall DATATBLS_CubeMainOutputParser(D2CubeOutputItem* pCubeOutputParam
 	{
 		pCubeOutputParam->nType = CUBEOP_COWPORTAL;
 	}
+#ifdef D2_VERSION_111_UBERS
```

Comment:
Please add the address of this function for the versions other than 1.10

// 1.11: D2Common.0x6FDAD210
// 1.14d: 0x668A90